### ISSUE TITLE
inputobject?

### ISSUE THREAD

**Issue by gwblok** [MicrosoftDocs/sccm-docs-powershell-ref#168]
[Enter feedback here]

Hey Team, could you provide an example. I've been trying all sorts of combinations today and can't get this to work.
It keeps asking for an inputobject, but I'm trying to create a rule, why is there an inputobject?  
I'm trying to make an application, then a rule using this command, then assign this rule to the application DT.  

Example of what I'm trying:
$NewRule = new-CMRequirementRuleOperatingSystemLanguageValue -RuleOperator OneOf -Culture "fr"

When I hit return, it prompts for inputobject.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ac5a1cbc-701c-03ca-c9e1-143e1cc7bda2
* Version Independent ID: 8a48993d-6f13-7b20-274e-e08efceff6b7
* Content: [New-CMRequirementRuleOperatingSystemLanguageValue (ConfigurationManager)](https://docs.microsoft.com/en-us/powershell/module/configurationmanager/new-cmrequirementruleoperatingsystemlanguagevalue?view=sccm-ps)
* Content Source: [sccm-ps/ConfigurationManager/New-CMRequirementRuleOperatingSystemLanguageValue.md](https://github.com/MicrosoftDocs/sccm-docs-powershell-ref/blob/master/sccm-ps/ConfigurationManager/New-CMRequirementRuleOperatingSystemLanguageValue.md)
* Product: **configuration-manager**
* Technology: **configmgr-other**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by aczechowski** [comment]
This will be fixed along with #158, see my recent comments there. specifically, see [1810 release notes](https://docs.microsoft.com/powershell/sccm/1810-release-notes#add-a-language-rule) which has some examples. (The very examples I incorporated back into the articles.

The inputobject for these req rule cmdlets is a global condition. Think of the GC as the template for the rule, and you use this cmdlet to create a specific instance of the template with the specific values. 

here's the specific example that this article will soon include (once published):

```powershell
$myGC = Get-CMGlobalCondition -Name "Operating System Language" | Where-Object PlatformType -eq 1

$cultureA = [System.Globalization.CultureInfo]::GetCultures([System.Globalization.CultureTypes]::AllCultures) | Where-Object Name -eq "ga-IE"

$cultureB = [System.Globalization.CultureInfo]::GetCultures([System.Globalization.CultureTypes]::AllCultures) | Where-Object Name -eq "hu-HU"

$myRule = $myGC | New-CMRequirementRuleOperatingSystemLanguageValue -RuleOperator OneOf -Culture $cultureA,$cultureB -IsMobile $False

Set-CMScriptDeploymentType -ApplicationName "Central App" -DeploymentTypeName "Install" -AddRequirement $myRule
```

Let me know if you have any questions. I really understand these cmdlets now!

**Comment by aczechowski** [comment]
I'm going to close this issue as it'll be addressed at the same time as #158.
